HISTORY.rst

```diff
@@ -2,6 +2,20 @@
 History
 =======
 
+1.2.0 (2020-06-05)
+------------------
+
+* Changed public interface of the learners to be more in line with the
+  scikit-learn interface (ongoing).
```

Comment:
```suggestion
  scikit-learn interface (ongoing). As part of these changes, it is no longer required to explicitly pass the data dimensionality to the learners on initialization.
```

I think we should mention this part separately, since it directly affects users and I think its quite the nice UX improvement.